content/docs/authorization/deployment/powerscale/_index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: PowerScale
+linktitle: PowerScale
+description: >
+  Enabling CSM Authorization for PowerScale CSI Driver
+---
+
+## Configuring PowerScale CSI Driver with CSM for Authorization
+
+Given a setup where Kubernetes, a storage system, and the CSM for Authorization Proxy Server are deployed, follow the steps below to configure the CSI Drivers to work with the Authorization sidecar:
```

Comment:
Change "the steps below" to "these steps"

Reply:
fixed.